users/stdiscosrv.rst

```diff
@@ -346,7 +346,25 @@ page. Note that that page is directed at setting up a proxy for the
 Syncthing web UI. You should do the proper path and port adjustments to proxying
 the discovery server and your particular setup.
 
+Traefik
+""""""
+
+Docker-Compose, must use -http option when starting server. You do not need to provide the ID parameter when adding the discovery server to the Syncthing client.
```

Comment:
What does "docker-compose" mean here by itself? Doesn't passing ID or not depend on the presented certificate, so this assumes a LetsEncrypt setup or similar?